feature-libs/organization/administration/components/unit/links/users/services/current-unit-user.service.ts

```diff
@@ -25,7 +25,7 @@ export class CurrentUnitUserService extends CurrentItemService<B2BUser> {
   }
 
   protected getItem(customerId: string): Observable<B2BUser> {
-    return customerId ? this.b2bUserService.get(customerId) : of({});
```

Comment:
I can't find the reason of that, it seems as called only in current-item.service
```
  readonly item$: Observable<T> = this.key$.pipe(
    switchMap((code: string) => (code ? this.getItem(code) : of(null)))
  );
```